ReactNativeTraining/src/redux/appNavigator/reducer.js

```diff
@@ -24,3 +24,12 @@ export const contacts = (state = [], action) => {
       return state;
   }
 };
+
+export const chat = (state = [], action) => {
+  switch (action.type) {
+    case 'CHAT_SELECTED':
+      return action.payload;
+    default:
+      return state;
+  }
+};
```

Comment:
Why we put all the reducers together? We had them separated in folders each one. Like this..
```
redux
   |-->messages 
   |        |---> actions / index.js
   |        |---> reducer / index.js
   |--->groups      
   |        |---> actions / index.js
   |        |---> reducer / index.js
....
```